### ISSUE TITLE
Semicolon is placed at a wrong place

### ISSUE THREAD

**Issue by Cosifne** [dotnet/roslyn#35598]
**Version Used**:  16.1.0 Preivew 1

**Steps to Reproduce**:
```using System;
using System.Collections.Generic;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace UnitTestProject7
{
    [TestClass]
    public class UnitTest1
    {
        private List<int> l = new List<int>()

        [TestMethod]
        public void TestMethod1()
        {
        }
    }
}

When I trying to type semicolon at the end of "private List<int> l = new List<int>()",
then the semicolon appears at the end of "[TestMethod]".
The semicolon should be placed at the end of "private List<int> l = new List<int>()".

**Comment by Cosifne** [comment]
FYI @chborl

**Comment by chborl** [comment]
@Cosifne , this is similar to #34666 and is fixed in 16.1 Preview 2 and up.  :)

**Comment by Cosifne** [comment]
> 
> 
> @Cosifne , this is similar to #34666 and is fixed in 16.1 Preview 2 and up. :)

👍 I'll close this issue then

**Comment by miloush** [comment]
Is this different then?

```C#
namespace Test16Bugs
{
    public class Program
    {
        public int MyProperty { get; set; }

        static void Main()
        {
            Main(new Program
            {
                MyProperty = 4|
            });
        }

        static void Main(Program p) { }
    }
}
```

Typing semicolon after `4` results in `; ;` at the end of the method call.

VisualStudio.16.Preview/16.1.0-pre.2.0+28822.285
